src/timer/SDL_timer.c

```diff
@@ -28,6 +28,8 @@
 
 /* #define DEBUG_TIMERS */
 
+#if 0
```

Comment:
Or maybe `#if !defined(__EMSCRIPTEN__) || !SDL_THREADS_DISABLED` so that we still use the default it pthreads are enabled...